components/dashboard_modules/CurrentCapacity.tsx

```diff
@@ -49,7 +49,7 @@ const CurrentCapacity = () => {
           : cur_output != null &&
             installed_capacity_kw != null &&
             installed_capacity_kw != 0
-          ? (cur_output / installed_capacity_kw).toFixed(0) + '%'
+          ? ((100 * cur_output) / installed_capacity_kw).toFixed(2) + '%'
```

Comment:
Is cur output not in kW?

Reply:
it is but to get percentage we need to multiply by 100

Reply:
or we just get a cute lil decimal